src/OpenSSL/crypto.py

```diff
@@ -1712,11 +1712,18 @@ def __init__(self, store, certificate):
         self._store_ctx = _ffi.gc(store_ctx, _lib.X509_STORE_CTX_free)
         self._store = store
         self._cert = certificate
+        self._chain = _ffi.NULL
         # Make the store context available for use after instantiating this
         # class by initializing it now. Per testing, subsequent calls to
         # :meth:`_init` have no adverse affect.
         self._init()
 
+    def _add_chain(self, chain):
```

Comment:
This can be removed once the other comment is addressed.

Reply:
Done.